dbhydra/src/abstract_db.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, config_file="config.ini", db_details=None):
         
         self.active_transactions=[]
         self.last_table_inserted_into: Optional[str] = None
+        self.identifier_quote = ''
```

Comment:
isn't this empty string?

Reply:
Yes, specified in the AbstractDb so it does not break concrete classes which do not have their own `self.identifier_quote`  but rely on methods inherited from `AbstractDb` where `self.identifier_quote` is used.